disallowed-insecure-calls.neon

```diff
@@ -67,3 +67,21 @@ parameters:
 		-
 			function: 'mysqli::real_query()'
 			message: 'use PDO::prepare() with variable binding/parametrized queries to prevent SQL Injection vulnerability'
+		-
+			function: 'rand()'
+			message: 'not cryptographically secure, use random_int() instead'
+		-
+			function: 'srand()'
+			message: 'not cryptographically secure, use random_int() instead'
```

Comment:
Although I see calling this function as a red flag, I'd leave this out, the function itself doesn't really do anything insecure.

Reply:
Removed